Header header

float32 steering_angle # the steering angle of the car in [rad]
